### PR TITLE
docs: fix perlls link

### PR DESCRIPTION
**Pull request by Z5483**

### CODE REVIEW

**Comment by mjlbach** [comment]
CONFIG.md is auto-generated, you need to edit the source file.

**Comment by Z5483** [comment]
> CONFIG.md is auto-generated, you need to edit the source file.

Sorry, I'm not familiar with this repository.
Was the latest change good?

File: lua/lspconfig/perlls.lua [hunk 1]
```diff
@@ -23,12 +23,12 @@ configs.perlls = {
   docs = {
     package_json = "https://raw.githubusercontent.com/richterger/Perl-LanguageServer/master/clients/vscode/perl/package.json";
     description = [[
-    https://github.com/richterger/Perl-LanguageServer/tree/master/clients/vscode/perl
+https://github.com/richterger/Perl-LanguageServer/tree/master/clients/vscode/perl
```
Comment (mjlbach) [inline lua/lspconfig/perlls.lua:26]: These links looks the same to me. What is the change, just the indentation?

Comment (Z5483) [reply lua/lspconfig/perlls.lua:26]: yeah, I forgot to change the PR title. Initially, the link was in a codeblock in `CONFIG.md` so I couldn't click on it.

**Comment by mjlbach** [comment]
Thanks for noticing :)

**Pull request merged**